### PR TITLE
Create Add iFARM as Collateral

### PR DESCRIPTION
**Pull request by EAsportz**

### CODE REVIEW

File: Add iFARM as Collateral [hunk 1]
```diff
@@ -0,0 +1,40 @@
+## HEADERS
```
Comment (mhairimca) [inline Add iFARM as Collateral:1]: Formatting needs adjustment

File: Add iFARM as Collateral [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+iFARM is the interest-bearing receipt of the cash flow token FARM from Harvest Finance. All FARM staked in the SP pool either directly or through iFARM receives the benefit of a portion of all profits across all Harvest vault strategies.  The Harvest community is interested in utilizing their capital more efficiently and borrowing against their iFARM while their FARM is still earning and exploring additional use cases within the UMA ecosystem such as KPI options.
+
+# TECHNICAL SPECIFICATIONS
+
+To accomplish this, two changes need to be made:
+
+* The iFARM address, 0x1571eD0bed4D987fe2b498DdBaE7DFA19519F651, needs to be added to the collateral currency whitelist introduced in UMIP-8
+
+* A final fee of 2 iFARM need to be added for the iFARM in the Store contract (~342.64 at time of writing).  Note: iFARM increases in value over time relative to FARM due to the nature of the in wallet compounding.
```
Comment (mhairimca) [inline Add iFARM as Collateral:24]: This is low, approx $316 as of 14th May,

Suggest 2.5 iFarm

**Comment by smb2796** [comment]
Hmm am I missing something here? iFARM was added as a supported collateral type in [UMIP-88](https://github.com/UMAprotocol/UMIPs/blob/master/UMIPs/umip-88.md). Can this be closed?

**Comment by smb2796** [comment]
Closing as this has already been voted in! Let me know if there was a specific reason for this proposal and we can reopen

**Pull request closed without merging**